### PR TITLE
Engagement page additions

### PR DESCRIPTION
**Pull request by twilliams24**
## Overview of changes
Cutting down some of the content on this page, and adding subheadings/neatening links to make the page easier to navigate.

## Checklist before requesting a review
- [x] I have checked the contributing guidelines
- [x] I have checked for and linked any relevant issues that this may resolve
- [x] I have checked that these changes build locally
- [x] I understand that if merged into main, these changes will be publicly available

### CODE REVIEW

File: statistics-production/user-eng.qmd [hunk 1]
```diff
@@ -140,54 +178,58 @@ DfE are working with ONS and others to consider how to take forward work in this
...
+
+In addition, teams should also think about the ways they can glean feedback on their publication(s), this may include internal checks regards enhancements such as participating in [content design scrum(s)](scrums.html).
 
 ---
 
 ## Consultations
 
 
-If you are considering changes to your publication and are contemplating a formal consultation do get in touch with HOP office early on to discuss.
+If you are considering changes to your publication and are contemplating a formal consultation do get in touch with [HOP office](mailto:Hop.statistics@education.gov.uk) early on to discuss.
```
Comment (cjrace) [inline statistics-production/user-eng.qmd:192]: Was already there, but should it be 'Statistics HoP office' for the link text?

File: statistics-production/user-eng.qmd [hunk 2]
```diff
@@ -140,54 +178,58 @@ DfE are working with ONS and others to consider how to take forward work in this
...
 
-To read more about consultations for background information please see the following two links: 
+For background information about consultation, please see:
+* [Code of Practice on Consultations](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/100807/file47158.pdf) - please bear in mind this was written back in 2008 so certain aspects may have been updated but the background and principles do remain the same.
+* [Consultation principles: guidance](https://www.gov.uk/government/publications/consultation-principles-guidance)
```
Comment (cjrace) [inline statistics-production/user-eng.qmd:196]: I think you're intending for these to be bullet points? Not sure the formatting has worked, here's what I see when building locally

image

Comment (cjrace) [reply statistics-production/user-eng.qmd:196]: Might need an additional empty line before the bullets start?

File: statistics-production/user-eng.qmd [hunk 3]
```diff
@@ -10,29 +10,31 @@
...
 
 
-Download this [top tips document](../resources/2021_05_14_Top_Tips.docx){target="_blank" rel="noopener noreferrer"} produced for the GSS, as a starting point for understanding how you can improve your user engagement.
+Download this [top tips document](../resources/2021_05_14_Top_Tips.docx){target="_blank" rel="noopener noreferrer"} as a starting point for understanding how you can improve your user engagement.
 
-Through our work engaging users on EES we have built up a list of interested users who have stated a willingness to be contacted about opportunities to be involved in user testing of our statistics on the platform, if you have any suggestions for things for us to test on EES with these users, please contact [explore.statistics@education.gov.uk](mailto:explore.statistics@education.gov.uk).
+If you have any suggestions of things for us to test on EES with users who have volunteered to do user testing, please contact [explore.statistics@education.gov.uk](mailto:explore.statistics@education.gov.uk).
 
 More advice and guidance will follow in this space. In the meantime, if you're interested in user engagement please contact Heather Brown.
```
Comment (cjrace) [inline statistics-production/user-eng.qmd:39]: should we just merge this last line with the one above and point them to the EES mailbox as Heather has left DfE now?

**Review by cjrace — CHANGES_REQUESTED** [review]

File: statistics-production/user-eng.qmd (hunk 1 above)
Comment (twilliams24) [reply statistics-production/user-eng.qmd]: have just added that in - good spot!

File: statistics-production/user-eng.qmd (hunk 2 above)
Comment (twilliams24) [reply statistics-production/user-eng.qmd:197]: good spot thanks! Have added that in now

File: statistics-production/user-eng.qmd (hunk 3 above)
Comment (twilliams24) [reply statistics-production/user-eng.qmd]: done!

**Review by cjrace — APPROVED** [review]

**Pull request merged**